connect/runtime/src/test/java/org/apache/kafka/connect/runtime/health/ConnectClusterStateImplTest.java

```diff
@@ -38,21 +40,25 @@
 
 @RunWith(PowerMockRunner.class)
 public class ConnectClusterStateImplTest {
-  
+    protected static final String KAFKA_CLUSTER_ID = "franzwashere";
+
     @Mock
     protected Herder herder;
     protected ConnectClusterStateImpl connectClusterState;
-    protected Collection<String> expectedConnectors;
```

Comment:
I'd keep it as member field. Having to declare the type in every test case to which you use it is be a bit annoying. Also, if you reuse this variable in consecutively in the same test case, the fact that in the first occurrence you have to specify the type but in the next ones it's already declared, breaks a symmetry in assignments that's otherwise good. Finally, having it `final` is not all that useful.